Uncaught (in promise) ReferenceError: process is not defined

Hi, first of all, thanks for great lib

[These lines ](https://github.com/rileytomasek/openai-fetch/blob/96682b69d7ea208b9cf8111d7860f2a4cf473ba3/src/openai-client.ts#L61-L62 )would cause an error in environments where process is not available (ex. Cloudflare or browsers) in case if `apiKey` or `organizationId` is not specified

Error I get when trying to run in Cloudflare worker 
```log
✘ [ERROR] Uncaught (in promise) ReferenceError: process is not defined

  const organizationId = opts.organizationId || process.env.OPENAI_ORG_ID;
                                                ^
      at OpenAIClient
  (/root/dev/app/node_modules/openai-fetch/src/openai-client.ts:62:50)
      at fetch (/root/dev/app/src/worker.ts:32:17)
```

As lib claims it supports all environments, including browsers, `process` should not be used as fallback

--- a/src/openai-client.ts
+++ b/src/openai-client.ts
@@ -58,6 +58,7 @@ export class OpenAIClient {
   api: ReturnType<typeof createApiInstance>;
 
   constructor(opts: ConfigOpts = {}) {
+    const process = globalThis.process || { env: {} };
     const apiKey = opts.apiKey || process.env.OPENAI_API_KEY;
     const organizationId = opts.organizationId || process.env.OPENAI_ORG_ID;
     if (!apiKey)

